C-Plus-Plus/Bit--Manipulation/Position_of_the_only_set_bit.cpp

```diff
@@ -0,0 +1,43 @@
+/*Find the position of the only set bit
+The main principle of this logic is one by one right shift the set bit of given number n until n becomes 0.
+Count how many times we shifted to make n zero. So final count is position of set bit.*/
+
+#include<bits/stdc++.h>
+using namespace std;
+int findPosition(int n){
+    int count=0;
+    int position=0;
+    while(n!=0){
+        if(n&1){
+            count++;
+        }
+        n=n>>1;   //Using right shift operator
```

Comment:
You need to add comments to your code. Make sure the comments you add are above the line of code and not alongside it.